src/Applications/LDAS_App/process_hist.csh

```diff
@@ -54,6 +54,13 @@ endif
 if($NENS > 1) then
    set GridComp = ENSAVG
    sed -i 's|VEGDYN|'VEGDYN0000'|g' $HISTRC
+   sed -i 's|CATCHCNCLM45|'CATCHCNCLM450000'|g' $HISTRC
```

Comment:
@weiyuan-jiang, @gmao-jkolassa, a couple of things here:
1. Why is there not a similar line for CATCHCNCLM4[0]?  Does the CatchCN ens run only work for CatchCNCLM45?
2. When we decided on CatchCN GridComp and restart names ending in numbers (as in catchcn45), we didn't think ahead to ensemble simulation.  It's rather ugly to have restarts with names such as "catchcnclm450018".  I'm afraid it would be rather disruptive to include a symbol to separate the model version from the ensemble id, as in "catchcnclm45_0000", but it may still be better to make the change now.  It's only going to get more disruptive.  An open question would be if the same change should be applied to "catch" restarts.  I'm not sure what to think.

Reply:
> @weiyuan-jiang, @gmao-jkolassa, a couple of things here:
> 
> 1. Why is there not a similar line for CATCHCNCLM4[0]?  Does the CatchCN ens run only work for CatchCNCLM45?
> 2. When we decided on CatchCN GridComp and restart names ending in numbers (as in catchcn45), we didn't think ahead to ensemble simulation.  It's rather ugly to have restarts with names such as "catchcnclm450018".  I'm afraid it would be rather disruptive to include a symbol to separate the model version from the ensemble id, as in "catchcnclm45_0000", but it may still be better to make the change now.  It's only going to get more disruptive.  An open question would be if the same change should be applied to "catch" restarts.  I'm not sure what to think.
> 3. For Nens>1, why should all catchcnclm45 exports come from ens member 0000?  (Maybe/probably I'm missing something here.)

Rolf,

1) I think nobody is using cnclm40 now.
2)This is unusual to export a variable from ensemble member. In general, we can get it from ENSAVG grid comp.
3)I think there is only one from ens member 0000, all others are from ENSAVG. Even this one can be moved to ENSAVG

Reply:
Good morning,

`I think nobody is using cnclm40 now.`

That is true. However, if it is not too much trouble I think it would make sense to also add the ensemble capability for CNCLM40. Otherwise, I think it could get very confusing in the future as to why different versions of CatchCN have different functionalities.

` I'm afraid it would be rather disruptive to include a symbol to separate the model version from the ensemble id, as in "catchcnclm45_0000", but it may still be better to make the change now. It's only going to get more disruptive.`

I agree that it would make sense to make this change now. @weiyuan-jiang I think Rolf was referring to the restart files that are created by ldas_setup in /input/restart. They currently will be called catchcnclm45XXXX (with XXXX being the ensemble ID) and catchcnclm45_XXXX would be cleaner. But I am not sure how much work that would actually entail.

`I think there is only one from ens member 0000, all others are from ENSAVG. Even this one can be moved to ENSAVG`

Yes, only CNFROOTC is coming from ensemble member 0000. I think it is handled differently because it is a CNCLM45-specific variable (that is, it does not exist in CNCLM40), so I don't know to what extent that impacts our ability to have it come from ENSAVG like the other variables.

Reply:
For my original items 1. and 3., see my comment below https://github.com/GEOS-ESM/GEOSldas/pull/584#discussion_r979076273

For item 2., we still need to understand better what it takes to change the CatchCN restart names for ensemble simulations from catchcnclm4[5]XXXX to catchcnclm4[5]_XXXX.  Having the CatchCN version number blend seamlessly into the ens id is clearly undesirable.  Maybe this can be limited to CatchCN ens simulations and doesn't have to touch Catch at all and CatchCN single-member runs.  Then there should be no impact on existing experiments and restart names.  But that would create an asymmetry between Catch and CatchCN ensemble simulations.  We can discuss more next week.